package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "screwdriver-scm-router",
-  "version": "2.0.0",
+  "version": "6.0.0",
```

Comment:
why do we have to bum up 4 major versions ?

Reply:
this is just a cosmetic change
we've bumped major version before and never updated here

Reply:
https://www.npmjs.com/package/screwdriver-scm-router is 5.x.x